tests/integration/api_asset_test.py

```diff
@@ -244,3 +269,467 @@ def test_restore_path(
     assert restored.guid == api_path.guid
     assert restored.qualified_name == api_path.qualified_name
     assert restored.status == EntityStatus.ACTIVE
+
+
+@pytest.fixture(scope="module")
+def api_object(
+    client: AtlanClient, connection: Connection
+) -> Generator[APIObject, None, None]:
+    assert connection.qualified_name
+    to_create = APIObject.create(
+        name=API_OBJECT_NAME, connection_qualified_name=connection.qualified_name
+    )
+    response = client.asset.save(to_create)
+    result = response.assets_created(asset_type=APIObject)[0]
+    yield result
+    delete_asset(client, guid=result.guid, asset_type=APIObject)
```

Comment:
We should use `creator()` instead (ref: https://github.com/atlanhq/atlan-python/pull/403/files#r1800760792)